gnc/sub8_trajectory_generator/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(sub8_trajectory_generator)
+
+find_package(catkin REQUIRED COMPONENTS
+  geometry_msgs
+  roscpp
+  message_generation
+)
+
+## Boost 
```

Comment:
We'll keep all of the messages in the [sub8_messages](https://github.com/uf-mil/Sub8/tree/master/utils/sub8_msgs) package